crates/bevy_render/src/camera/projection.rs

```diff
@@ -134,54 +134,71 @@ impl Default for PerspectiveProjection {
     }
 }
 
-// TODO: make this a component instead of a property
-#[derive(Debug, Clone, Reflect, FromReflect, Serialize, Deserialize)]
-#[reflect(Serialize, Deserialize)]
-pub enum WindowOrigin {
-    Center,
-    BottomLeft,
-}
-
 #[derive(Debug, Clone, Reflect, FromReflect, Serialize, Deserialize)]
 #[reflect(Serialize, Deserialize)]
 pub enum ScalingMode {
-    /// Manually specify left/right/top/bottom values.
+    /// Manually specify the projection's size with `width` and `height`.
     /// Ignore window resizing; the image will stretch.
-    None,
-    /// Match the window size. 1 world unit = 1 pixel.
-    WindowSize,
-    /// Use minimal possible viewport size while keeping the aspect ratio.
+    None { width: f32, height: f32 },
+    /// Match the window size.
+    /// The argument is the number of pixels that equals one world unit.
+    WindowSize(f32),
+    /// Use the minimal possible projection size while keeping the aspect ratio.
     /// Arguments are in world units.
     Auto { min_width: f32, min_height: f32 },
-    /// Keep vertical axis constant; resize horizontal with aspect ratio.
-    /// The argument is the desired height of the viewport in world units.
+    /// Keep the projection's height constant; adjust width to match aspect ratio.
+    /// The argument is the desired height of the projection in world units.
     FixedVertical(f32),
-    /// Keep horizontal axis constant; resize vertical with aspect ratio.
-    /// The argument is the desired width of the viewport in world units.
+    /// Keep the projection's width constant; adjust height to match aspect ratio.
+    /// The argument is the desired width of the projection in world units.
     FixedHorizontal(f32),
 }
 
+/// Project a 3D space onto a 2D surface using parallel lines, i.e., unlike [`PerspectiveProjection`],
+/// the size at which objects appear remain the same regardless of depth.
+///
+/// The volume contained in the projection is called the *view frustum*. Since the viewport is rectangular,
+/// the view frustum is in the shape of a rectangular prism.
```

Comment:
Would the following be more precise?

> Since the viewport is rectangular, *and* the projection lines are parallel, the view frustum is in the shape of a rectangular prism?

If that is not correct, then I am still not clear on why the view frustum is a rectangular prism if the viewport is rectangular.

Reply:
Yep, will add this next commit, thanks!